### PR TITLE
Implemented token saving in background.js, added message listener to all...

### PR DESCRIPTION
**Pull request by rmagee88**
...ow retrieval

### CODE REVIEW

File: src/background/background.js [hunk 1]
```diff
@@ -1,10 +1,22 @@
 'use strict';
-var appState = 'off';
+var userToken;
 
+// Listener for messages coming to the background script
 chrome.extension.onMessage.addListener(function(request,sender,sendResponse) {
```
Comment (wettowelreactor) [inline src/background/background.js]: I believe chrome.extension has been depreciated for chrome.runtime.

Comment (rmagee88) [inline src/background/background.js]: changed!

-Rich

**Comment by wettowelreactor** [comment]
LGTM

**Comment by mrblueblue** [comment]
LGTM

**Pull request merged**